services/ux-backend/handlers/expandstorage/handler.go

```diff
@@ -268,7 +268,7 @@ func createCephFilesystemStorageClass(w http.ResponseWriter, r *http.Request, cl
 		Parameters: map[string]string{
 			"clusterID": namespace,
 			"fsName":    filesystemName,
-			"pool":      poolName,
+			"pool":      fmt.Sprintf("%s-%s", filesystemName, poolName),
```

Comment:
is pool name mandatory when we expand the storage? if not Rook appends different value with integers

Reply:
Yaa in the UI the pool name is mandatory,
cc @SanjalKatiyar to keep me honest

Reply:
in that case lets make it mandatory here as well

Reply:
yeah, UI has it as a mandatory field.